src/redux/books/booksSlice.js

```diff
@@ -1,43 +1,84 @@
-import { createSlice } from '@reduxjs/toolkit';
-
-const booksData = [
-  {
-    item_id: 'item1',
-    title: 'The Great Gatsby',
-    author: 'John Smith',
-    category: 'Fiction',
-  },
-  {
-    item_id: 'item2',
-    title: 'Anna Karenina',
-    author: 'Leo Tolstoy',
-    category: 'Fiction',
-  },
-  {
-    item_id: 'item3',
-    title: 'The Selfish Gene',
-    author: 'Richard Dawkins',
-    category: 'Nonfiction',
-  },
-];
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const apiUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/aDqZ6eCZp1GfaZZvdWPr/books';
+
+export const apiData = createAsyncThunk('books/apiData', async () => {
+  try {
+    const response = await axios.get(apiUrl);
+    return response.data;
+  } catch (error) {
+    throw new Error('Error al obtener los datos de la API');
+  }
+});
+
+export const apiPost = createAsyncThunk('books/apiPost', async (newBook) => {
+  try {
+    const response = await axios.post(apiUrl, newBook);
```

Comment:
**You have done a great job!**
One more step, to get approved! 

- [ ] Your 'GET' requiest is working perfectly. The issue is with post request. Each time if I try to add a new book, it shows me `403` error.  Please fix the issue and make sure it's posting as expected. Possible solution woule be with your api key. Try to update your api key and check if it's posting correctly.

<img width="555" alt="Screenshot 2023-07-21 at 09 28 10" src="https://github.com/Dachrono/Bookstore/assets/69011963/f94a160e-2afb-4ada-8162-1f0094f2e9af">

Reply:
Solved